src/pages/components/anchor.mdx

```diff
@@ -63,6 +63,17 @@ import AnchorExternalCode from '!!raw-loader!../../examples/anchor/AnchorExterna
   <AnchorExternal />
 </CodeExample>
 
+#### Security
+
+When linking to third-party content you may be [exposing your site to performance and security issues](https://web.dev/external-anchors-use-rel-noopener/).
```

Comment:
Seems pretty unfortunate that a description of external links, with an external URL, isn't being properly formatted with an external link treatment. Perhaps this gets sorted when markdown anchors are wrapped Gatsby [`Link`](https://github.com/zendeskgarden/website/blob/962427a3b36640a811501593b9f04e5b701a0dc1/src/layouts/Root/components/StyledNavigationLink.tsx#L20-L36) components.